### PR TITLE
Feature/us8392 card carousel

### PR DESCRIPTION
**Pull request by zarahzachz**
Add functionality for a carousel on the card deck. On mobile screens, cards should scroll and on desktop it should remove the carousel and allow cards to wrap to new lines.

---
Refactor the card carousel for mobile to be smoother/less janky.

This is currently implemented on the /live/stream page and will be used for the "happening at crossroads" section on the homepage. 

Unclear how this solution will play with ng. 

Consider using this library... https://flickity.metafizzy.co/

### CODE REVIEW

File: src/examples/card-carousel/index.html [hunk 1]
```diff
@@ -0,0 +1,140 @@
...
+          <h4 class="card-title card-title--overlap text-uppercase">Goon Docks</h4>
+          <div class="card-text">
+            <address>
+              1632 Doubloon Cove<br>
+              Astoria, OR 19840 (<a href="#">Map</a>)
+            </address>
+
+            <strong>Service Times:</strong>
+            <ul class="list-unstyled">
+              <li>SAT 4:30 & 6:15pm;</li>
```
Comment (seancdavis) [inline src/examples/card-carousel/index.html]: Does `&` cause an error in any browsers today? I know, at least in the past, the preferred markup for an ampersand was `&amp;`.

File: src/examples/card-carousel/layout.js [hunk 2]
```diff
@@ -0,0 +1,35 @@
+function carousel_styles() {
+  const carousel = document.body.querySelector('.card-deck--carousel');
```
Comment (seancdavis) [inline src/examples/card-carousel/layout.js]: Querying elements in this way leads to [this](https://developer.mozilla.org/en-US/docs/Web/API/Document/querySelector):

> Returns the first Element within the document that matches the specified selector, or group of selectors.

This means that only the first matched element will be returned. That's problematic if there are more than one on a page.

Not sure exactly how, but you should probably account for multiple carousels on a page.

File: src/examples/card-carousel/layout.js [hunk 3]
```diff
@@ -0,0 +1,35 @@
+function carousel_styles() {
```
Comment (seancdavis) [inline src/examples/card-carousel/layout.js]: Global function is problematic, but we can talk more about this and determine a good path forward.

File: src/examples/card-carousel/index.html (hunk 1 above)
Comment (zarahzachz) [inline src/examples/card-carousel/index.html]: Ah yes - I don't think this effects modern browsers anymore, but I'll go ahead and address it.

**Pull request merged**